app/lib/fake/server/fake_server_entrypoint.dart

```diff
@@ -66,14 +68,18 @@ class FakeServerCommand extends Command {
     final dataFile = argResults!['data-file'] as String?;
     final watch = argResults!['watch'] == true;
 
-    Logger.root.onRecord.listen((r) {
-      print([
-        r.time.toIso8601String(),
-        r.toString(),
-        r.error,
-        r.stackTrace?.toString(),
-      ].where((e) => e != null).join(' '));
-    });
+    if (envConfig.isInsideCI) {
+      setupDebugEnvBasedLogging(extraDebug: 'fake_server pub.email');
+    } else {
+      Logger.root.onRecord.listen((r) {
+        print([
+          r.time.toIso8601String(),
+          r.toString(),
+          r.error,
+          r.stackTrace?.toString(),
+        ].where((e) => e != null).join(' '));
+      });
```

Comment:
Why do we need two different options here?

Can we just specify the `DEBUG` env variable in CI?

Reply:
Removed the custom one, only using the single debug-based flow.